src/olympia/addons/serializers.py

```diff
@@ -300,7 +300,7 @@ def fake_version_object(self, obj, data):
             version = Version(
                 addon=obj, id=data['id'],
                 reviewed=self.handle_date(data['reviewed']),
-                version=data['version'])
+                version=data['version'], channel=data['channel'])
```

Comment:
Ah, sure.
The existing tests were testing the url was matching what `version.get_url_path` was supposed to return (they were failing, which led to this fix) but I'll explicitly test the url output too.